StartBonusActivationOrchestrator/handler.ts

```diff
@@ -82,8 +83,10 @@ export const getStartBonusActivationOrchestratorHandler = (
         name: "bonus.activation.success"
       });
     } else {
-      yield context.df.callActivity(
+      // TODO: is FailedBonusActivationActivity idempotent?
```

Comment:
cc @balanza

Reply:
what's needed to state this?

Reply:
`FailedBonusActivationActivity` use queries for update/delete data that don't throw errors. Only relaseLockForUserFamily does. Do we want a retry even on other queries failures?

Reply:
yes, it looks to me it's idempotent. what do you think about it?